python/ray/worker.py

```diff
@@ -865,6 +924,10 @@ def _process_task(self, task):
         function_name = self.function_execution_info[self.task_driver_id.id()][
             function_id.id()].function_name
 
+        if not self.serialization_context_map.has_key(self.task_driver_id):
```

Comment:
`has_key` is removed in python 3.
we should use `self.task_driver_id not in self.serialization_context_map` instead.